probe-rs/src/flashing/flash_algorithm.rs

```diff
@@ -176,6 +174,15 @@ impl FlashAlgorithm {
         0x0477_0D1F,
     ];
 
+    /// When the target architecture is not known, and we need to allocate space for the header,
+    /// this function returns the maximum size of the header of supported architectures.
+    pub fn get_max_algorithm_header_size() -> usize {
```

Comment:
We should make this u64 as we never use this as an usize.

Reply:
Great catch!